app/views/curiosity_cards/index.html.erb

```diff
@@ -0,0 +1,60 @@
+<div class="row colums is-mobile">
+  <div class="column is-2 is-offset-10">
+    <%= link_to 'Add New Curiosity',
+      new_curiosity_card_path, class: 'button is-info mb-2 is-outlined' %>
+  </div>
+</div>
+<div class="row columns is-multiline" id="app">
+  <% @curiosities.each do |curiosity| %>
+    <div class="column is-3">
+      <div class="card">
+        <div class="card-image">
+          <figure class="image is-16by9">
+            <% if curiosity.images.attached? %>
+              <%= image_tag(url_for(curiosity.images[0]) ) %>
```

Comment:
You shouldn't worry about this right now, but FYI, you're displaying images as they were uploaded. That means if there are 10 images 4K web browser would be downloading ~20-30MB
You can use a [variant](https://edgeguides.rubyonrails.org/active_storage_overview.html#attaching-files-to-records) to display a thumbnail instead
<img width="1503" alt="Screen Shot 2022-01-21 at 8 10 35" src="https://user-images.githubusercontent.com/26292224/150541728-aa4d2251-8ef4-42b0-b97c-cdddd60142d6.png">